lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2077,14 +2077,43 @@ ProcessModule(ModuleSP module_sp, std::string m_description,
   module_search_paths.insert(module_search_paths.end(),
                              opts.getImportSearchPaths().begin(),
                              opts.getImportSearchPaths().end());
-  for (const auto &fwsp : opts.getFrameworkSearchPaths())
-    framework_search_paths.push_back({fwsp.Path, fwsp.IsSystem});
   auto &clang_opts = invocation.getClangImporterOptions().ExtraArgs;
   for (const std::string &arg : clang_opts) {
     extra_clang_args.push_back(arg);
     LOG_VERBOSE_PRINTF(GetLog(LLDBLog::Types), "adding Clang argument \"%s\".",
                        arg.c_str());
   }
+  // Unfortunately, this is insufficient, as ClangImporter can
```

Comment:
This reads like a comment followed by some commented out code.  It would read a lot more clearly if you did:

// Unfortunately, this:
//     for (const)...
//         framework...
// is insufficient, as...